lib/influxdb/client/http.rb

```diff
@@ -4,15 +4,16 @@
 require 'net/https'
 
 module InfluxDB
+  # rubocop:disable Metrics/ModuleLength
   # rubocop:disable Metrics/MethodLength
   # rubocop:disable Metrics/AbcSize
   module HTTP # :nodoc:
-    def get(url, options = {})
+    def get(url, options = {}, json_streaming = false)
```

Comment:
Please make `json_streaming` a key of the `options` hash. We don't need the value here, but only in `handle_successful_response`.